specs/Deployment/DeploymentAPI.md

```diff
@@ -179,12 +179,57 @@ they shut down, to refer to the updated framework package.
     }
 ```
 
+#### OnError_*
+
+Additional behavior is available if an error occurs in `DeploymentManager.Initialize()`,
+per the following logic:
+
+```c++
+DeploymentManager::Initialize(...)
+{
+    HRESULT hr = S_OK
+
+    // Only supported in packaged processes
+    if IsPackagedProcess()
+    {
+        // Do the initialization work
+        hr = _Initialize(...)
+    }
+    else
+    {
+        // We're not a packaged process. Is calling Initialize an error or ignore it?
+        if options.OnNoPackageIdentity_NOOP
+            return OK
+        else
+            hr = HRESULT_FROM_WIN32(ERROR_NOT_SUPPORTED)
+    }
+    if FAILED(hr)
+    {
+        LogToEventLog(hr, ...)
+
+        // Should we pop the debugger?
+        if options.OnError_DebugBreak or (options.OnError_DebugBreak_IfDebuggerAttached and IsDebuggerPresent())
```

Comment:
This looks like C++ but isn't. Should this instead point to the implementation? This code-oriented description of the behavior is hard to follow.

Feels like there should be a sample of showing the app setting these flags.

Reply:
RECOMMNED: Revise per comment above